docs/server/push-duty.rst

```diff
@@ -63,6 +59,30 @@ The tag is pushed to production by Jason, once approved by Krupa, on Thursdays.
 It is the responsibility of the push hero to follow-up with Krupa and Jason,
 and be around during the push for any unexpected issues.
 
+Monitoring the push
++++++++++++++++++++
+
+The best places to monitor the results of the push are:
+
+* `Sentry <http://sentry.mktmon.services.phx1.mozilla.com/mkt/addonsmozillaorg/>`_
+* `Graphite <http://dashboard.mktadm.ops.services.phx1.mozilla.com/graphite?site=addons&graph=all-responses>`_
+
+.. note:: Both of these URLs require you to be logged into Mozilla VPN.
+
+Graphite should show no large spikes or changes. Sentry should show no new errors, although
+it will show intermittent errors and the occasional error as the push occurs.
+
+Once complete
++++++++++++++
+
+Create a new etherpad template that contains blocking bugs and the like, for example:
```

Comment:
Should it say something about "create the new etherpad *for the next push*" or something?